packages/common/fetchWrappers.tsx

```diff
@@ -3,14 +3,17 @@ interface DataProps {
 }
 
 export const post = (url: string, data: DataProps, options = {} as { [key: string]: any }) => {
-  const { optionHeaders, ...otherOptions } = options
+  const { headers: optionHeaders, ...otherOptions } = options
```

Comment:
this wasn't working — always returning a 400 error. 
so I borrowed the implementation from www/lib/fetchWrapper.ts

this function is only used twice, here and in feedback submission in docs. Checked and that still works.